### PR TITLE
ica_apply_unfiltered - new function using MNE type of pre-whitening

### PR DESCRIPTION
**Pull request by jdammers**
@ckiefer0 @pravsripad @fboers for discussion:  I added a new simple function to apply ICA (based on filtered data) to unfiltered raw data. This could serve as a replacement for ica_replace_mean_std.py which is based on a copy of the MNE ICA class (old version which is not updated).

### CODE REVIEW

**Comment by jdammers** [comment]
The advantage is that changes in the ICA class (done by MNE) will be recognized. Later we need to add more functionality by means of allow other than raw instances. In case we want to use the jumeg style of prewhitening, I recommend to use the fastica or informax routines in jumeg directly.

File: jumeg/decompose/ica.py [hunk 1]
```diff
@@ -794,3 +794,27 @@ def ica2data_single_components(sources, ica, pca, picks=None):
...
+
+
+#######################################################
+#                                                     #
+# apply ICA based on filtered data to unfiltered raw  #
+#                                                     #
+#######################################################
+def ica_apply_unfiltered(raw_unfilt, ica_filt, picks):
+
+    data = raw_unfilt.get_data(picks, None, None, None)
```
Comment (pravsripad) [inline jumeg/decompose/ica.py:806]: It would be useful to expose the start, stop parameters here for use when applying ICA on chops.

**Comment by pravsripad** [comment]
Works on jumeg sample data as expected. 

image

**Comment by pravsripad** [comment]
I like the idea of using this to keep up to date with MNE. But since this does not replace the jumeg style of whitening (i.e. calculating std on per channel basis), do we still keep the  ica_replace_mean_std.py?

File: jumeg/decompose/ica.py (hunk 1 above)
Comment (jdammers) [reply jumeg/decompose/ica.py:806]: @pravsripad Thanks, this can easily be added to the function

**Comment by jdammers** [comment]
This is something we should discuss soon. My suggestion is to use FastICA or Infomax as implemented in Jumeg instead (both ICA version use the jumeg style of 'prewhitening')  What's is missing (or not finished yet) is an instance of the MNE-type of ICA object. Currently the ICA oject is a copy of the sklearn ICA instance.

**Pull request merged**